Tuto-GUDHI-cover-complex.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>I am facing an error on this cell:</p><pre class="ql-syntax" spellcheck="false">Need coordinates for running differential coordinates!
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
/tmp/ipykernel_44266/2760060715.py in &lt;module&gt;
----&gt; 1 cover_complex.compute_differential_coordinates(nodes=bnd[1])

nerve_gic.pyx in gudhi.nerve_gic.CoverComplex.compute_differential_coordinates()

RuntimeError: No active exception to reraise

</pre><p>For information, I have (with cover_complex.bootstrap_topological_features(10) - 100 may be  a bit too much):</p><pre class="ql-syntax" spellcheck="false">bnd[1] = [16, 21, 4, 9]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>